subsys/bluetooth/host/conn.c

```diff
@@ -410,6 +412,28 @@ static struct bt_conn_tx *conn_tx_alloc(void)
 	return k_fifo_get(&free_tx, K_FOREVER);
 }
 
+int bt_conn_send_iso_cb(struct bt_conn *conn, struct net_buf *buf,
+			bt_conn_tx_cb_t cb, bool has_ts)
+{
+	int err = bt_conn_send_cb(conn, buf, cb, NULL);
+
+	if (err) {
+		return err;
+	}
+
+	/* We need a tx context to do the op below. We always allocate one if
+	 * `cb` is set.
+	 */
+	__ASSERT_NO_MSG(cb);
```

Comment:
Isn't it better to check for the stated requirement?

```suggestion
	__ASSERT_NO_MSG(tx_data(buf)->tx);
```

Also, the comment does not make sense to me. What is the "the op below"? (Preemptive argument: It seems to me that the new code in this PR only cares about `tx_data(buf)->iso_has_ts`. That is valid memory even if `tx_data(buf)->tx` is `NULL`, so the requirement commented/asserted here is not true, right?)